infrastructure/event/eventtest/taskqueue.go

```diff
@@ -0,0 +1,27 @@
+package eventtest
+
+import (
+	"github.com/utahta/momoclo-channel/domain/core"
+	"github.com/utahta/momoclo-channel/domain/event"
+)
+
+// TaskQueue presents mock
+type TaskQueue struct {
+	log   core.Logger
+	Tasks []event.Task
+}
+
+// NewTaskQueue returns no ops taskQueue instance
+func NewTaskQueue(log core.Logger) *TaskQueue {
+	return &TaskQueue{log, nil}
+}
+
+func (t *TaskQueue) Push(task event.Task) error {
+	t.Tasks = append(t.Tasks, task)
+	return nil
+}
+
+func (t *TaskQueue) PushMulti(tasks []event.Task) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method TaskQueue.PushMulti should have comment or be unexported